src/raven.js

```diff
@@ -1821,6 +1859,56 @@ function htmlElementAsString(elem) {
     return out.join('');
 }
 
+
+function isOnlyOneTruthy(a, b) {
+    return a && !b || b && !a;
```

Comment:
I'd prefer this alternative function body:
```javascript
return !!(!!a ^ !!b);
```
The xor operator makes it more obvious what's going on IMO, and this will *always* return a boolean; current version has some edge cases where it won't like `isOnlyOneTruthy('', '')`.